components/brave_news/common/pref_names.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
I wonder why this is not treated as a rename? Any idea how we can improve it, e.g. via rename then modify? And then using a merge commit for this PR?

Reply:
I guess the file contents changed too much for Git to deal with :/ We could do the merge but honestly, I don't know if it's worth it - git is a bit hopeless with renames and it should be fairly obvious what's going on from the commit name.

If this is breaking CI I think I'd be happier just updating the copyright header, tbh

Reply:
It's less what's going on from the commit name and more when someone wants to figure out the history of the pref names and the trail stops at this commit. But it's ok if it's just a couple files out of the whole PR.